examples/medplum-nextauth-demo/package.json

```diff
@@ -25,7 +25,7 @@
     "@mantine/notifications": "7.17.2",
     "@medplum/core": "4.0.3",
     "@medplum/eslint-config": "4.0.3",
-    "next": "15.2.2",
+    "next": "15.2.3",
```

Comment:
The important update which addresses: https://github.com/medplum/medplum/security/dependabot/225